L1Trigger/L1CaloTrigger/python/Phase1L1TJets_9x9_cff.py

```diff
@@ -1,25 +1,25 @@
 import FWCore.ParameterSet.Config as cms
 from math import pi
 
-from L1Trigger.L1CaloTrigger.Phase1L1TJetProducer_cfi import Phase1L1TJetProducer
-from L1Trigger.L1CaloTrigger.Phase1L1TJetCalibrator_9x9Jets_cfi import Phase1L1TJetCalibrator as Phase1L1TJetCalibrator9x9
-from L1Trigger.L1CaloTrigger.Phase1L1TJetSumsProducer_cfi import Phase1L1TJetSumsProducer
+from L1Trigger.L1CaloTrigger.Phase1L1TJetProducer_cfi import l1tPhase1JetProducer
+from L1Trigger.L1CaloTrigger.Phase1L1TJetCalibrator_9x9Jets_cfi import l1tPhase1JetCalibrator as l1tPhase1JetCalibrator9x9
+from L1Trigger.L1CaloTrigger.Phase1L1TJetSumsProducer_cfi import l1tPhase1JetSumsProducer
 
-Phase1L1TJetProducer9x9 = Phase1L1TJetProducer.clone(
+l1tPhase1JetProducer9x9 = l1tPhase1JetProducer.clone(
 	  jetIEtaSize = 9,
 	  jetIPhiSize = 9,
 	  outputCollectionName = "UncalibratedPhase1L1TJetFromPfCandidates"
 )
 
-Phase1L1TJetCalibrator9x9.inputCollectionTag = ("Phase1L1TJetProducer9x9", "UncalibratedPhase1L1TJetFromPfCandidates", "")
-Phase1L1TJetCalibrator9x9.outputCollectionName = ("Phase1L1TJetFromPfCandidates")
+l1tPhase1JetCalibrator9x9.inputCollectionTag = ("l1tPhase1JetProducer9x9", "UncalibratedPhase1L1TJetFromPfCandidates", "")
```

Comment:
Why not cloning also this one, instead of modifying the original config?

Reply:
This creates conflicts because l1tPhase1JetProducer is imported from several cfi (same name in several cfi but different parameters)

Reply:
> This creates conflicts because l1tPhase1JetProducer is imported from several cfi (same name in several cfi but different parameters)

This is exactly what I would like to avoid also for `l1tPhase1JetCalibrator` (to which the comment refers)

If you want to modify `l1tPhase1JetCalibrator` you must do it in L1Trigger.L1CaloTrigger.Phase1L1TJetCalibrator_9x9Jets_cfi
Here you should clone `l1tPhase1JetCalibrator9x9` from `l1tPhase1JetCalibrator` and update only the local copy